src/us/kbase/workspace/test/workspace/WorkspaceTest.java

```diff
@@ -5509,9 +5552,10 @@ public void setGlobalRead() throws Exception {
 	
 	@Test
 	public void hiddenObjects() throws Exception {
-		WorkspaceUser user = new WorkspaceUser("hideObjUser");
-		WorkspaceIdentifier wsi = new WorkspaceIdentifier("hideObj");
-		WorkspaceUser user2 = new WorkspaceUser("hideObjUser2");
+		final WorkspaceUser user = new WorkspaceUser("hideObjUser");
```

Comment:
holy ..., is this file ever going to end?!

Reply:
yeah it's bad. A relic of my test writing abilities 10 years ago.

Reply:
FWIW https://github.com/kbase/workspace_deluxe/blob/master/src/us/kbase/workspace/test/workspace/WorkspaceTest.java#L100-L118